_data/locales/ar.yml

```diff
@@ -1,29 +1,30 @@
 ar:
   locale_name: العربية
-  subtitle: مدير الحزم الذي يفتقده macOS
+  subtitle: مدير الحزم الذي يفتقده macOS (أو Linux)
   pagecontent:
-    question: مالذي سيفعله Homebrew ؟
```

Comment:
Why remove the existing translations?

Reply:
Hey @MikeMcQuaid, I took a copy of the latest en.yml and I'm going to translate it again and refer to the old translation.

Do you have a different approach in mind?

Reply:
Ok, cool. Was just confused, thanks.